CareKit/CareCard/OCKCareCardButton.m

```diff
@@ -53,6 +55,16 @@ - (void)drawRect:(CGRect)rect {
         [[UIColor clearColor] setFill];
         UIRectFill(_circleLayer.frame);
     }
+    if (!_label) {
+        CGRect labelRect = CGRectMake(0, ButtonSize/2 + 8, ButtonSize, ButtonSize);
+        _label = [[UILabel alloc] initWithFrame:labelRect];
+        _label.font = [UIFont systemFontOfSize:9 weight:300];
```

Comment:
Can we use any UIKIT_EXTERN const for a UIFontWeight instead of 300. I think 'heavy' looks good.